docs/using_eessi/building_on_eessi.md

```diff
@@ -0,0 +1,69 @@
+# Building software on top of EESSI
+
+## Building software on top of EESSI with EasyBuild
+Building on top of EESSI with EasyBuild is relatively straightforward. One crucial feature is that EasyBuild supports building against operating system libraries that are _not_ in a standard prefix (such as `/usr/lib`). This is required when building against EESSI, since all of the software in EESSI is build against the [compatibility layer](../compatibility_layer.md).
+
+### Starting the EESSI software environment
+Start your environment as described [here](../using_eessi/setting_up_environment.md)
+
+### Configure EasyBuild
+To configure EasyBuild, first, check out the Github repository. We advice you check out the branch corresponding to the version of EESSI you would like to use.
+
+If you are unsure which version you are using, you can run
+```
+echo ${EESSI_PILOT_VERSION}
+```
+to check it.
+
+To build on top of e.g. version `2023.06` of the EESSI software stack, we check it out, and go into that directory:
+
+```
+git clone https://github.com/EESSI/software-layer/ --branch 2023.06
+cd software-layer
+```
+Then, you have to pick a working directory (that you have write access to) where EasyBuild can do the build, and an install directory, where EasyBuild can install it. In this example, we use `/tmp/easybuild` as our working directory, and `$HOME/.local/easybuild` as our installpath:
+```
+export WORKDIR=/tmp/easybuild
```

Comment:
Should we use something similar to what is suggested in #117? For example,
```bash
export WORKDIR=$(mktemp --directory --tmpdir=/tmp  -t eessi.XXXXXXXXXX)
```
Although, `eessi` should probably be replaced with something else, e.g., `build`, to make it easier to distinguish different temp directories from each other ?

Reply:
done, will call it `eessi-build.XXXXXXXXX`